chromium_src/chrome/browser/ui/views/incognito_clear_browsing_data_dialog.cc

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
+
+#define SetShowCloseButton(SHOW) SetShowCloseButton(false)
```

Comment:
1. vars should use `lower_case_underscore`, but I think you can just write `#define SetShowCloseButton(...) SetShowCloseButton(false)`.
2. pls add `#undef`:
```
...
#include "src/chrome/browser/ui/views/incognito_clear_browsing_data_dialog.cc"

#undef SetShowCloseButton
```

Reply:
Hi @goodov, I have updated accordingly and seems like it's working fine 
Please have a look, thank you 🙏